src/golang/cmd/server/routes/routes.go

```diff
@@ -24,6 +24,7 @@ const (
 	NodeDagOperatorsRoute          = "/api/v2/workflow/{workflowID}/dag/{dagID}/node/operators"
 	NodeOperatorContentRoute       = "/api/v2/workflow/{workflowID}/dag/{dagID}/node/operator/{nodeID}/content"
 	NodesResultsRoute              = "/api/v2/workflow/{workflowID}/result/{dagResultID}/nodes/results"
+	GetServerEnvironmentRoute      = "/api/v2/environment"
```

Comment:
We will have one route handle multiple methods in v2 moving forward. So it's sufficient to call it `EnvironmentRoute`.